apps/core-angular-cli/angular.json

```diff
@@ -25,9 +25,10 @@
             "aot": true,
             "assets": ["src/favicon.ico", "src/assets"],
             "styles": [
-              "src/styles.scss",
+              "node_modules/normalize.css/normalize.css",
               "node_modules/@clr/core/global.min.css",
-              "node_modules/@clr/city/css/bundles/default.min.css"
+              "node_modules/@clr/city/css/bundles/default.min.css",
+              "src/styles.scss"
```

Comment:
Looks like `styles.scss` file is empty now. Do we need to keep this file?

Reply:
I left this one since its the default from the Angular CLI, I didnt want to change to much of the defaults so its easy to update in the future and easier to follow as an example